docs/developer-docs/integrations/sns/introduction/sns-intro-high-level.md

```diff
@@ -7,9 +7,7 @@ sidebar_position: 1
 A Service Nervous Systems (SNS) is an advanced form of a decentralized autonomous organization (DAO).
 It is assumed that if you are reading this page you know what a DAO is and that you want to learn how
 you can hand over your dapp to an SNS DAO.
-If this is not (yet) the case, you can learn more about DAOs and SNSs
-[here](../tokenomics/index.md),
-including what advantages such decentralized governance systems have.
+If this is not (yet) the case, please refer to the [SNS page](https://internetcomputer.org/sns) or the [high level introduction of how SNSs work](../../../../../how-it-works/3_governance/sns.subpage.md).
```

Comment:
@olaszakos is there a better way to refer to https://internetcomputer.org/sns ? 
I wasn't sure in which directory to find it..